src/agent_fork/worktree_list.py

```
@@ -0,0 +1,38 @@
+def parse_worktree_list(output: bytes) -> tuple[WorktreeRecord, ...]:
+    """Parse porcelain records, flushing the final open record.
+
+    Paths are always resolved. Every caller compares them against resolved
+    paths, and the one historical unresolved comparison — in ``verify_fork``,
+    which matched a resolved ``creation.path`` against a raw porcelain string —
+    was a defect rather than a supported mode.
+    """
+    records: list[WorktreeRecord] = []
+    path: str | None = None
+    branch: str | None = None
+    for line in output.decode(errors="surrogateescape").splitlines() + [""]:
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)